freqtrade/main.py

```diff
@@ -179,7 +179,7 @@ def handle_trade(trade: Trade) -> bool:
 
     logger.debug('Handling %s ...', trade)
     current_rate = exchange.get_ticker(trade.pair)['bid']
-    if should_sell(trade, current_rate, datetime.utcnow()):
```

Comment:
`get_signal` returns a bool not an int, you can drop the `== 1`.
I think we should rename `should_sell` into `min_roi_reached()` or something like that, because should_sell is too board in this case.

Reply:
Yes, agreed with the naming change.